scanner/src/funTest/kotlin/scanners/scancode/ScanCodeScannerFunTest.kt

```diff
@@ -36,16 +36,20 @@ class ScanCodeScannerFunTest : AbstractScannerFunTest(setOf(ExpensiveTag, ScanCo
     override val expectedFileLicenses = setOf("Apache-2.0".toSpdx())
     override val expectedDirectoryLicenses = setOf("Apache-2.0".toSpdx())
 
+    val licenseTextDirs = listOfNotNull(scanner.licenseTextDir)
+
     init {
         "return the full license text for the HERE proprietary license" {
-            val text = getLicenseText("LicenseRef-scancode-here-proprietary")?.trim()
+            val text =
+                getLicenseText("LicenseRef-scancode-here-proprietary", licenseTextDirectories = licenseTextDirs)?.trim()
```

Comment:
The problem is that we'd need to add this extra argument to *every* call of `getLicenseText()` throughout the code base, not just to calls from this test...

Reply:
Yes.  However, I am not sure whether these other places truly need this extra license directory. 

And I am not sure how to create a proper test environment so that I can *fully* test my changes *locally*.

Reply:
> I am not sure whether these other places truly need this extra license directory.

They do. E.g. in the reporters, we need to be able to get the text for ScanCode-specific licenses.

> And I am not sure how to create a proper test environment so that I can fully test my changes locally.

I'm currently (for different reasons) experimenting with migrating fully to GitHub actions, away from Azure pipelines, and using https://github.com/nektos/act to run the actions locally. Maybe that would also work for you.

Reply:
When running github actions through https://github.com/nektos/act, I finally got this error:
```
| Execution failed for task ':clients:github-graphql:graphqlGenerateClient'.
| > There was a failure while executing work items
|    > A failure occurred while executing com.expediagroup.graphql.plugin.gradle.actions.GenerateClientAction
|       > java.nio.charset.MalformedInputException: Input length = 1
```

I am using `ghcr.io/catthehacker/ubuntu:act-latest`.

@sschuberth Any suggestion?

Reply:
IIRC this is a side-effect of `act` not supporting any kind of caching actions, and applying the necessary work-arounds is a bit messy, also see my https://github.com/nektos/act/issues/989. That's actually why I gave up on using `act` for now in favor of finishing up https://github.com/oss-review-toolkit/ort/pull/5031. I've already merged a few related changes over the last days, and I believe I could be ready by tomorrow.

Reply:
> IIRC this is a side-effect of act not supporting any kind of caching actions, and applying the necessary work-arounds is a bit messy, also see my nektos/act#989. 

I made some changes to `.github/workflows/build-and-test.yml` as you suggested in nektos/act#989 to work around cache issue.

> That's actually why I gave up on using act for now in favor of finishing up #5031. I've already merged a few related changes over the last days, and I believe I could be ready by tomorrow.

Looking forward to that.

Now I am running these tasks manually, and find a few issues that need to be fixed.